tests/unit/python/foglamp/common/test_statistics.py

```diff
@@ -123,54 +138,59 @@ def test_load_keys(self):
-        s = Statistics(storage_client_mock)
+        s = statistics.Statistics(storage_client_mock)
 
         with pytest.raises(exception_name) as excinfo:
             await s.update(key, value_increment)
         assert exception_message == str(excinfo.value)
 
     async def test_update_exception(self):
         storage_client_mock = MagicMock(spec=StorageClient)
-        s = Statistics(storage_client_mock)
+        s = statistics.Statistics(storage_client_mock)
         msg = 'Unable to update statistics value based on statistics_key %s and value_increment %d,' \
               ' error %s', 'BUFFERED', 5, ''
         with patch.object(s._storage, 'update_tbl', side_effect=Exception()):
             with pytest.raises(Exception):
-                with patch.object(_logger, 'exception') as logger_exception:
+                with patch.object(statistics._logger, 'exception') as logger_exception:
                     await s.update('BUFFERED', 5)
             logger_exception.assert_called_once_with(*msg)
 
     async def test_add_update(self):
         stat_dict = {'FOGBENCH/TEMPERATURE': 1}
         storage_client_mock = MagicMock(spec=StorageClient)
-        s = Statistics(storage_client_mock)
+        s = statistics.Statistics(storage_client_mock)
         payload = '{"where": {"column": "key", "condition": "=", "value": "FOGBENCH/TEMPERATURE"}, ' \
                   '"expressions": [{"column": "value", "operator": "+", "value": 1}]}'
-        expected_result = {"response": "updated", "rows_affected": 1}
-        with patch.object(s._storage, 'update_tbl', return_value=expected_result) as stat_update:
+        async def mock_coro():
+            return {"response": "updated", "rows_affected": 1}
+
+        with patch.object(s._storage, 'update_tbl', return_value=mock_coro()) as stat_update:
             await s.add_update(stat_dict)
-            assert expected_result['response'] == "updated"
         stat_update.assert_called_once_with('statistics', payload)
 
     async def test_insert_when_key_error(self):
         stat_dict = {'FOGBENCH/TEMPERATURE': 1}
         storage_client_mock = MagicMock(spec=StorageClient)
-        s = Statistics(storage_client_mock)
-        with patch.object(_logger, 'exception') as logger_exception:
-            with pytest.raises(KeyError):
-                await s.add_update(stat_dict)
+        s = statistics.Statistics(storage_client_mock)
+        async def mock_coro():
+            return {"response": "not updated", "rows_affected": 0}
+
+        with patch.object(s._storage, 'update_tbl', return_value=mock_coro()) as stat_update:
+            with patch.object(statistics._logger, 'exception') as logger_exception:
+                with pytest.raises(KeyError):
+                    await s.add_update(stat_dict)
         args, kwargs = logger_exception.call_args
         assert args[0] == 'Statistics key %s has not been registered'
         assert args[1] == 'FOGBENCH/TEMPERATURE'
```

Comment:
Fix the indentation of logger_exception assertions. Also add `stat_update` assertions